README.md

```diff
@@ -223,6 +224,106 @@ To avoid the confusion between connection, stream, transport, and other names th
 - connection - something that implements the transversal expectations of a stream between two peers, including the benefits of using a stream plus having a way to do half duplex, full duplex
 - transport - something that as a dial/listen interface and return objs that implement a connection interface
 
+### Stats API
```

Comment:
Do you see this all RED too? 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/1211152/36469950-fc2d05cc-16e0-11e8-8b27-c22d9df03f97.png">

Mind checking what went wrong here?